### DISCUSSION TITLE
Not able to install safe exam browser in windows

### DISCUSSION THREAD

**Discussion by Bekescaba809** [SafeExamBrowser/seb-win-refactoring#688] [Q&A]
I am not able to install safe exam browser in windows.It gives below error
<img width="386" alt="Screenshot 2023-10-22 184458" src="https://github.com/SafeExamBrowser/seb-win-refactoring/assets/148694840/dc0a394e-352e-4a8d-a1d5-9be0dfa68fea">
[Safe_Exam_Browser_20231022181241.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/13063343/Safe_Exam_Browser_20231022181241.log)
please help

**Comment by dbuechel** [comment]
Please do not post the same issue twice. We will not process it more rapidly nor give it more attention just because you post it multiple times.
